strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/validation/groupId/GroupIdCaseValidator.java

```diff
@@ -0,0 +1,37 @@
+package org.carlspring.strongbox.storage.validation.groupId;
+
+import org.carlspring.strongbox.artifact.coordinates.MavenArtifactCoordinates;
+import org.carlspring.strongbox.providers.ProviderImplementationException;
+import org.carlspring.strongbox.providers.io.RepositoryFileAttributes;
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.nio.file.Files;
+
+/**
+ * Created by dinesh on 12/6/17.
+ */
+@Component("GroupIdCaseValidator")
+public class GroupIdCaseValidator implements GroupIdValidator {
```

Comment:
Some notes:
* I believe a better name for this class would be MavenGroupIdLowercaseValidator.
* Could you please follow the coding convention described [here](https://github.com/strongbox/strongbox/wiki/Coding-Convention)?